README.rst

```diff
@@ -65,7 +63,7 @@ The modifications in this fork are minor fixes and cosmetic changes, such as:
 * JUnit XML output format
 * Overriding repository root auto-detection via --repository
 * Support ``#pragma once`` as an alternative to header include guards
-* ... and a few more (most of which are open PRs on upstream)
+* ... and `quite a bit <https://github.com/cpplint/cpplint/blob/master/changelog.rst>`_ more
```

Comment:
this should point to the `develop` branch

Reply:
When we change the branches to the same one, I feel like we should name that branch master. Plus, we'd merge all released changes in develop to master anyway.